x-pack/plugins/apm/server/ui_settings.ts

```diff
@@ -8,25 +8,24 @@ import { schema } from '@kbn/config-schema';
 import { i18n } from '@kbn/i18n';
 import { UiSettingsParams } from '../../../../src/core/types';
 import {
-  enableSignificantTerms,
+  enableCorrelations,
   enableServiceOverview,
 } from '../common/ui_settings_keys';
 
 /**
  * uiSettings definitions for APM.
  */
 export const uiSettings: Record<string, UiSettingsParams<boolean>> = {
-  [enableSignificantTerms]: {
+  [enableCorrelations]: {
     category: ['observability'],
     name: i18n.translate('xpack.apm.enableCorrelationsExperimentName', {
-      defaultMessage: 'APM Significant terms (Platinum required)',
+      defaultMessage: 'APM Correlations (Platinum required)',
```

Comment:
I don't think this should be capitalized.

Reply:
i retained the same capitalization pattern as before, but i agree it doesn't need to be capitalized here